planner/core/src/main/java/eu/seaclouds/platform/planner/core/HttpHelper.java

```diff
@@ -83,22 +84,48 @@ public String getRequest(String restPath, List<NameValuePair> params){
         }
     }
 
+    public String postInBody(String restPath, String bodyContent){
+        HttpPost post = new HttpPost(prepareRequestURL(restPath));
+        try {
+            StringEntity entity = new StringEntity(bodyContent);
+            post.setEntity(entity);
+            HttpResponse response = httpclient.execute(post);
+            String result = EntityUtils.toString(response.getEntity());
+            return result;
+        }catch (ClientProtocolException e) {
+            return null;
```

Comment:
@szenzaro : At several points of the code, you are returning null at several points when an exception is caught. If you return (instead of rethrowing the exception), the exception must be logged. Otherwise, you won't know what is happening, even when testing the application. 

This hasn't to do with having a error-handling policy or not.